### PR TITLE
Create maven project

### PR DESCRIPTION
**Pull request by SibingWu**
As per title, create a maven project for "Walk-in Interview"

### CODE REVIEW

File: walk-in-interview/pom.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>walk-in-interview</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+
+</project>
```
Comment (htthach) [inline walk-in-interview/pom.xml]: New line at EOF?

File: walk-in-interview/pom.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>walk-in-interview</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
```
Comment (htthach) [inline walk-in-interview/pom.xml:10]: Maybe remove extra blank line?

**Review by riya-n — APPROVED** [review]

File: walk-in-interview/pom.xml (hunk 1 above)
Comment (SibingWu) [reply walk-in-interview/pom.xml]: Fixed

File: walk-in-interview/pom.xml (hunk 2 above)
Comment (SibingWu) [reply walk-in-interview/pom.xml:10]: Fixed

**Review by htthach — APPROVED** [review]

**Pull request merged**